src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -343,7 +342,56 @@ namespace winrt::TerminalApp::implementation
         {
             auto top = heap.top();
             heap.pop();
-            _filteredActions.Append(top.command);
+            actions.Append(top.command);
+        }
+    }
+
+    // Method Description:
+    // - Update our list of filtered actions to reflect the current contents of
+    //   the input box. For more details on which commands will be displayed,
+    //   see `_getWeight`.
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - <none>
+    void CommandPalette::_updateFilteredActions()
+    {
+        auto actions = winrt::single_threaded_observable_vector<winrt::TerminalApp::Command>();
```

Comment:
nit: the `single_threaded_observable_vector` is probably overkill here, since I don't think anyone's going to be listening for changes on this buffer 😉. We could probably get away with a `winrt::single_threaded_vector` (or heck, even just a `std::vector`) here

Reply:
Indeed that was overkill, and a mere std::vector is the perfect amount of kill